templates/whatson/event_page.html

```diff
@@ -8,4 +8,25 @@
 
 {% block content %}
     {% include "includes/event_hero.html" with title=page.title image=page.lead_image event_type=page.event_type access_descriptor=page.primary_access_type %}
+
+
+    {# Hosts and speakers #}
+    <div class="tna-container">
```

Comment:
@janehughes you may need to change these container divs when bringing this into the overall page layout